README.md

```diff
@@ -1 +1,28 @@
-# Discord Bot
+
+# Proton
+
+Ever wanted a Bot to help you manage your Server? Well the answer is Proton
+
+It has all the features you want online user tracking or welcome and goodbye messages

```

Comment:
i think you need to work on your marketing lol

Reply:
Maybe we should hire someone from your university who studies that field lmao